handler_test.go

```diff
@@ -670,9 +666,9 @@ func TestSessionDisabled(t *testing.T) {
 	client := quotacontrol.NewClient(logger, Service, cfg, nil)
 
 	options := &authcontrol.Options{
-		JWTSecret: Secret,
-		KeyFuncs:  []authcontrol.KeyFunc{middleware.KeyFromHeader},
-		UserStore: server.Store,
+		JWTSecret:      Secret,
+		AccessKeyFuncs: []authcontrol.AccessKeyFunc{authcontrol.AccessKeyFromHeader},
```

Comment:
I left this here, just as example how you can set this explicitly as well.. but this is the default